.cirrus.yml

```diff
@@ -646,8 +668,12 @@ verify_test_built_images_task:
     env:
         ADD_SECOND_PARTITION: 'true'
         matrix:
-            TEST_REMOTE_CLIENT: 'true'
-            TEST_REMOTE_CLIENT: 'false'
+            - name: remote
+              env:
+                  TEST_REMOTE_CLIENT: 'true'
+            - name: local
+              env:
+                  TEST_REMOTE_CLIENT: 'false'
         matrix:
             # Required env. var. by check_image_script
             PACKER_BUILDER_NAME: "fedora-30"
```

Comment:
It would reduce duplication if we used your new `$FEDORA_NAME` variable here for the value, and similar for the lines below.

Reply:
Good catch, thanks